src/scripts.js

```diff
@@ -65,37 +74,85 @@ const populateBookingArea = () => {
   })
 }
 
-//On login button click, invoke function to fetch
-//all data, pass in argument as user number
-
-
-
 
 // ***** ON WINDOW LOAD *****
 window.addEventListener('load', () => {
-    hideAll([topHalf, bottomHalf])
+    hideAll([topHalf, bottomHalf, roomsAvailablePage])
 })
 
 
 // ***** LOGIN PAGE *****
-checkDateBtn.addEventListener('click', () => {
-    checkDate()
+checkDateBtn.addEventListener('click', (event) => {
+  event.preventDefault()
+  roomsAvailableSection.innerHTML = ''
+    if (selectRoomType.value === 'All options') {
```

Comment:
This is a great way to inject HTML into the DOM, but I am wondering if you can make it slightly more dynamic? Right now it writes to the DOM when 'All options' is selected - is there a way to just have a variable called 'options' that you update based on the filter options?